modules/shared_utils.py

```diff
@@ -11,6 +13,13 @@ class Owner(IntEnum):
     ZONE_0 = 0
     ZONE_1 = 1
 
+    @classmethod
+    def zones(cls) -> Set[Owner]:
```

Comment:
What's the purpose of this? Would `set(Owner)` not work?

Reply:
This is just a relic of the previous way of differentiating beacons, removed in e1c0e195ba84c3f54bed936f45e7b991144a2649.